cmd/kubeadm/app/phases/addons/manifests.go

```diff
@@ -0,0 +1,273 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package addons
+
+const (
```

Comment:
I thought we wanted to use dynamic kubelet config (https://github.com/kubernetes/kubernetes/pull/29459), iirc that is what was told to me when I wanted to do a similar change to using Go templates.